container/attributes.go

```diff
@@ -12,7 +12,66 @@ const (
 
 	// SysAttributeZone is a string of zone for container name.
 	SysAttributeZone = SysAttributePrefix + "ZONE"
+
+	// SysAttributeHomomorphicHashing is a container's homomorphic hashing state.
+	SysAttributeHomomorphicHashing = SysAttributePrefix + "HOMOMORPHIC-HASHING"
 )
 
 // SysAttributeZoneDefault is a default value for SysAttributeZone attribute.
 const SysAttributeZoneDefault = "container"
+
+const (
+	homomorphicHashingDisabledValue = "false"
+)
+
+// HomomorphicHashingDisabled returns container's homomorphic
+// hashing state:
+// 	* true, if hashing is disabled;
+// 	* false, if hashing is enabled.
+//
+// All container's attributes must be unique, otherwise behaviour
+// if undefined.
+//
+// See also SetHomomorphicHashingState.
+func (c Container) HomomorphicHashingDisabled() bool {
+	for i := range c.attr {
+		if c.attr[i].key == SysAttributeHomomorphicHashing {
+			return c.attr[i].val == homomorphicHashingDisabledValue
+		}
+	}
+
+	return false
+}
+
+// SetHomomorphicHashingState sets homomorphic hashing state for
+// container.
+//
+// All container's attributes must be unique, otherwise behaviour
+// if undefined.
+//
+// See also HomomorphicHashingDisabled.
+func (c *Container) SetHomomorphicHashingState(enable bool) {
+	for i := range c.attr {
+		if c.attr[i].key == SysAttributeHomomorphicHashing {
```

Comment:
```suggestion
		if c.attr[i].Key() == SysAttributeHomomorphicHashing {
```
since currently `Attribute` isn't an internal type of `Container`.